release-notes/8.0/preview/api-diff/preview4/Microsoft.NETCore.App/8.0-preview4_System.Runtime.Intrinsics.Wasm.md

```diff
@@ -0,0 +1,166 @@
+# System.Runtime.Intrinsics.Wasm
```

Comment:
`System.Runtime.Intrinsics.Wasm` changes LGTM

CC. @dotnet/area-system-runtime-intrinsics

Reply:
@radekdoulik, are these "on by default" now or is there still work before that happens?